src/DssSpell.t.sol

```diff
@@ -7,6 +7,30 @@ import "dss-interfaces/Interfaces.sol";
 
 contract DssSpellTest is DssSpellTestBase {
 
+    // Recognized Delegates Wallets
+    address immutable FLIPFLOPFLAP   = wallets.addr("FLIPFLOPFLAP");
+    address immutable ULTRASCHUPPI   = wallets.addr("ULTRASCHUPPI");
+    address immutable FEEDBLACKLOOPS = wallets.addr("FEEDBLACKLOOPS");
+    address immutable MAKERMAN       = wallets.addr("MAKERMAN");
+    address immutable ACREINVEST     = wallets.addr("ACREINVEST");
+    address immutable MONETSUPPLY    = wallets.addr("MONETSUPPLY");
+    address immutable JUSTINCASE     = wallets.addr("JUSTINCASE");
+    address immutable GFXLABS        = wallets.addr("GFXLABS");
+    address immutable DOO            = wallets.addr("DOO");
+    address immutable FLIPSIDE       = wallets.addr("FLIPSIDE");
+
+    // Recognized Delegates Payout
+    uint256 constant amountFlipFlopFlap  = 12_000;
+    uint256 constant amountUltraSchuppi  = 12_000;
+    uint256 constant amountFeedBlack     = 12_000;
+    uint256 constant amountMakerMan      = 10_929;
+    uint256 constant amountAcreInvest    =  9_347;
+    uint256 constant amountMonetSupply   =  8_626;
+    uint256 constant amountJustinCase    =  7_522;
+    uint256 constant amountGfxLabs       =  6_607;
+    uint256 constant amountDoo           =    351;
+    uint256 constant amountFlipside      =    265;
```

Comment:
I think it would be better to just avoid setting the immutables/constants and define all what is needed in the test.

Reply:
Does it really matter? this is the way it was done before and when moving it to the test we need to handle `stack too deep` errors. Let me know if you feel strong about it and if so I'll move and handle the errors.

Reply:
Well I was trying to put all together what next week needs to be commented/removed. But yes, the stack too deep will be a problem. Other option is putting the constants and auxiliar functions right before the test.
Anyway it's fine leaving this way as well. Not big deal at all.